lox_services/utils/enums.py

```diff
@@ -32,3 +32,28 @@ class Files(Enum):
     contract = "contract.csv"
     refunds_label_not_used = 'refunds_label_not_used.csv'
 
+
+
+class BQParameterType(str, Enum):
+    """
+    types of parameter which can be used in parameterized query.
+    """
+    #Sequence types
+    ARRAY="ARRAY"
```

Comment:
Please keep the spacing consistent. Sequence class constants in this class have no spaces, while scalar class constants do.